prow/config/config.go

```diff
@@ -69,12 +69,22 @@ type Config struct {
 
 	// PushGateway is a prometheus push gateway.
 	PushGateway PushGateway `json:"push_gateway,omitempty"`
+
+	// Owners contains configuration related to handling OWNERS files.
+	Owners Owners `json:"owners",omitempty"`
 }
 
 type PushGateway struct {
 	Endpoint string `json:"endpoint,omitempty"`
 }
 
+// Owners contains configuration related to handling OWNERS files.
+type Owners struct {
+	// MdYamlRepos is a list of org and org/repo strings specifying the repos that support yaml
+	// OWNERS file configs in markdown file headers.
+	MdYamlRepos []string `json:"mdyamlrepos",omitempty`
```

Comment:
I'm confused about this. Can you say more and give an example?

Also, general go rule is to keep things like MD and YAML consistently-cased.

Reply:
Fixed the casing, added an example, and tried to explain a little more clearly.

Basically you can put OWNERS-like configs into YAML headers in `.md` files, but it needs to be enabled on a per repo basis so that we dont have to open and parse every markdown file in all the repos (it's only used in the docs repo right now).